quic/s2n-quic/src/provider/event/console_perf.rs

```diff
@@ -0,0 +1,296 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::provider::event;
+use core::time::Duration;
+use std::sync::{
+    atomic::{AtomicU64, Ordering},
+    Arc,
+};
+
+/// An event subscriber that prints performance metrics to the console
```

Comment:
Probably good to call out somewhere where we're printing

```suggestion
/// An event subscriber that prints performance metrics to the console via STDOUT
```